scripts/add-helm-code.sh

```diff
@@ -21,13 +21,26 @@
 # Working directory does not matter.
 
 HOME_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )"
+# Directory containing the yaml file
+dir=${HOME_DIR}/chart/templates
 
-if [ "$#" -lt 1 ]; then
-    echo "adds/remove helm tags to cluster scoped files such as ClusterRole and CLusterRoleBinding"
-    echo "Usage: $(basename $0) add | remove"
-    exit
-fi
-CMD=$1
+while (( $# > 0 )); do
+	case "$1" in
+    (-h|--help)
+    	echo "adds/remove helm tags to cluster scoped files such as ClusterRole and CLusterRoleBinding"
```

Comment:
I don't know if "helm tags" is a thing. In any case, that is not what this script does. What this script does is hack the names of certain API objects, to make them specialize to the WDS.

Reply:
Again, this is beyond the scope of this change.  I don't mind updating the comment, but I have noticed that this scripts calls something named 'cleanup_helm_tags'.  I don't fill confident enough updating the comments without digging a bit deeper into what this script actually does.

Reply:
You're right, this PR did not introduce this problem.

Reply:
So this does not have to be fixed in this PR.